dm-wolftpm-wolfssl-tls-with-zephyr/src/user_settings.h

```diff
@@ -0,0 +1,511 @@
+#endif
+//#define HAVE_AES_DECRYPT
+#define WOLFSSL_AES_COUNTER
+#define WOLFSSL_AES_CFB
+//#define WOLFSSL_AES_OFB
+//#define HAVE_AESCCM
+//#define WOLFSSL_AES_XTS
+
+//#define NO_AES_128
+//#define NO_AES_192
+//#define NO_AES_256
+//#define WOLFSSL_AES_SMALL_TABLES
+//#define WOLFSSL_AES_NO_UNROLL
+
+
+/* HKDF */
+#if defined(WOLFSSL_TLS13) || defined(CONFIG_WOLFSSL_HKDF)
+    #define HAVE_HKDF
+#endif
+
+/* CMAC - Zephyr nRF BTLE needs CMAC */
+#if 1
+    #define WOLFSSL_AES_DIRECT
+    #define WOLFSSL_CMAC
+#endif
+
+
+
+
+/* Disable Algorithms */
+#define NO_DSA
+#define NO_RC4
+#define NO_MD4
+#define NO_MD5
+#define NO_DES3
+#define WOLFSSL_NO_SHAKE128
+#define WOLFSSL_NO_SHAKE256
+
+
+
+/* ------------------------------------------------------------------------- */
+/* Math */
+/* ------------------------------------------------------------------------- */
+/* Math Options */
+/* Multi-precision - generic math for all keys sizes and curves */
+#if 0
+    #define WOLFSSL_SP_MATH /* no multi-precision math, only single */
+#elif 1
+    /* wolf mp math (sp_int.c) */
+    #define WOLFSSL_SP_MATH_ALL /* use SP math for all key sizes and curves */
+    //#define WOLFSSL_SP_NO_MALLOC
+
+    /* use smaller version of code */
+    #define WOLFSSL_SP_SMALL
+
+    /* Define the maximum math bits used */
+    #if !defined(NO_RSA) || !defined(NO_DH)
+        #define SP_INT_BITS 2048
+    #elif defined(HAVE_ECC)
+        #define SP_INT_BITS 256
+    #endif
+
+#elif 1
+    /* Fast Math (tfm.c) (stack based and timing resistant) */
+    #define USE_FAST_MATH
+    #define TFM_TIMING_RESISTANT
+
+    /* Define the maximum math bits used (2 * max) */
+    #if !defined(NO_RSA) || !defined(NO_DH)
+        #define FP_MAX_BITS (2*2048)
+        #ifdef HAVE_ECC
+            #define ALT_ECC_SIZE /* use heap allocation for ECC point */
+        #endif
+    #elif defined(HAVE_ECC)
+        #define FP_MAX_BITS (2*256)
+    #endif
+    #ifdef HAVE_ECC
+        //#define TFM_ECC256 /* optional speedup for ECC-256 bit */
+    #endif
+#else
+    /* Normal (integer.c) (heap based, not timing resistant) - not recommended */
+    #define USE_INTEGER_HEAP_MATH
+#endif
+
+/* Single Precision (optional) */
+/* Math written for specific curves and key sizes */
+#if 1
+    #ifdef HAVE_ECC
+        #define WOLFSSL_HAVE_SP_ECC
+        //#define WOLFSSL_SP_NO_256
+        //#define WOLFSSL_SP_384
+        //#define WOLFSSL_SP_521
+    #endif
+    #ifndef NO_RSA
+        #define WOLFSSL_HAVE_SP_RSA
+        //#define WOLFSSL_SP_NO_2048
+        //#define WOLFSSL_SP_NO_3072
+        //#define WOLFSSL_SP_4096
+    #endif
+    #ifndef NO_DH
+        #define WOLFSSL_HAVE_SP_DH
+    #endif
+
+    #define WOLFSSL_SP_SMALL      /* use smaller version of code */
+    //#define WOLFSSL_SP_NO_MALLOC /* disable heap in wolf/SP math */
+    //#define SP_DIV_WORD_USE_DIV /* no div64 */
+
+    #define WOLFSSL_SP_ARM_CORTEX_M_ASM
+
+#if 1
+    #define WOLFSSL_ARMASM
```

Comment:
Perhaps do not enable this by default? Duplicated again at 396. Just move it down there.